pypicloud/cache/base.py

```diff
@@ -6,6 +6,7 @@
 from pyramid.settings import asbool
 
 import posixpath
+from flywheel.fields.types import UTC
```

Comment:
Since `flywheel` isn't a strict dependency of pypicloud (and I'd like to keep it that way), this import can fail on systems where only a `pip install pypicloud` has been run. I'd recommend just copying the UTC class (it was copied from [the python docs](https://docs.python.org/2/library/datetime.html) in the first place) and putting it into `util.py` or wherever.